### ISSUE TITLE
Check arguments in the generated publication

### ISSUE THREAD

**Issue by mquandalle** [matteodem/meteor-easy-search#328]
Hi @matteodem,

Thank you for the 2.0 update :-) I tried to update [Wekan](https://github.com/wekan/wekan) but unfortunately I ran into the following issue:

The MongoDB engine won’t work if one use the `audit-argument-checks` package because the publication generated by EasySearch doesn’t `check` the arguments it receives:

```
Exception from sub users/easySearch id aZxqNfJcQx72uYe3C Error: Did not check() all arguments during publisher 'users/easySearch'
```

The appropriate `check`s should be added here: https://github.com/matteodem/meteor-easy-search/blob/852a58cc2b30e893a04558865fb0ea47534e260a/packages/easysearch:core/lib/core/search-collection.js#L140

**Comment by dburles** [comment]
+1

**Comment by matteodem** [comment]
oh that's right, gonna have a look at this soon.

**Comment by matteodem** [comment]
Should be fixed in the latest version `2.0.2`
